test/extended/networking/commatrix.go

```diff
@@ -0,0 +1,215 @@
+package networking
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/openshift-kni/commatrix/pkg/client"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+	"github.com/openshift-kni/commatrix/pkg/consts"
+	"github.com/openshift-kni/commatrix/pkg/types"
+	"github.com/openshift-kni/commatrix/pkg/utils"
+	configv1client "github.com/openshift/client-go/config/clientset/versioned/typed/config/v1"
+	exutil "github.com/openshift/origin/test/extended/util"
+	"github.com/sirupsen/logrus"
+
+	commatrixcreator "github.com/openshift-kni/commatrix/pkg/commatrix-creator"
+	"github.com/openshift-kni/commatrix/pkg/endpointslices"
+
+	configv1 "github.com/openshift/api/config/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+var _ = g.Describe("[sig-network][Feature:commatrix][Serial]", func() {
+	g.It("should apply firewall by blocking all ports except the ones OCP is listening on", func() {
+		g.By("generating the commatrix")
+
+		cs, err := client.New()
+		if err != nil {
+			panic(fmt.Errorf("failed creating the k8s client: %w", err))
```

Comment:
It's a test, use o.Expect(err).NotTo(o.HaveOccurred()) instead of panic

Reply:
mistake